### PR TITLE
feat(api/applications): get representation response update

### PR DESCRIPTION
**Pull request by jonathangoulding**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/src/server/repositories/representation.repository.js [hunk 1]
```diff
@@ -126,9 +126,23 @@ export const getById = async (id, caseId) => {
 			},
 			representationActions: {
```
Comment (jonathangoulding) [inline apps/api/src/server/repositories/representation.repository.js:127]: @Philip-Philippou1 do we want to change this in the schema to workflow ?

Comment (Philip-Philippou1) [reply apps/api/src/server/repositories/representation.repository.js:127]: I think this is ok, representationActions is more descriptive in the DB and describes where the table belongs.

File: apps/api/src/server/applications/application/representations/representation.mapper.js [hunk 2]
```diff
@@ -85,3 +85,20 @@ export const mapCreateOrUpdateRepRequestToRepository = (
  */
 export const getLatestRedaction = ({ representationActions }) =>
```
Comment (Philip-Philippou1) [inline apps/api/src/server/applications/application/representations/representation.mapper.js:86]: I know this is existing code but can we guarantee this is a redaction and not a status update?

Comment (jonathangoulding) [reply apps/api/src/server/applications/application/representations/representation.mapper.js:86]: done - 

export const getLatestRedaction = ({ representationActions }) =>
	representationActions
		.filter(({ type }) => type === 'REDACTION')
		.slice(-1)
		.pop();

**Review by Philip-Philippou1 — APPROVED** [review]

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**